src/content/api/cli.md

```diff
@@ -44,47 +44,49 @@ webpack-cli offers a variety of commands to make working with webpack easy. By d
-| `--devtool, -d`     | string          | Controls if and how source maps are generated.                                                                 |
-| `--prefetch`        | string          | Prefetch this request                                                                                          |
-| `--json, -j`        | boolean, string | Prints result as JSON or store it in a file                                                                    |
-| `--mode`            | string          | Defines the mode to pass to webpack                                                                            |
-| `--version, -v`     | boolean         | Get current version                                                                                            |
-| `--stats`           | boolean, string | It instructs webpack on how to treat the stats                                                                 |
-| `--analyze`         | boolean         | It invokes `webpack-bundle-analyzer` plugin to get bundle information                                          |
+| Flag / Alias            | Type            | Description                                                                                                    |
+| ----------------------- | --------------- | -------------------------------------------------------------------------------------------------------------- |
+| `--entry`               | string[]        | The entry point(s) of your application e.g. `./src/main.js`                                                    |
+| `--config, -c`          | string[]        | Provide path to a webpack configuration file e.g. `./webpack.config.js`                                        |
+| `--config-name`         | string[]        | Name of the configuration to use                                                                               |
+| `--name`                | string          | Name of the configuration. Used when loading multiple configurations                                           |
+| `--color`               | boolean         | Enable colors on console                                                                                       |
+| `--merge, -m`           | boolean         | Merge two or more configurations using webpack-merge e.g. `-c ./webpack.config.js -c ./webpack.test.config.js` |
+| `--env`                 | string[]        | Environment passed to the configuration when it is a function                                                  |
+| `--progress`            | boolean, string | Print compilation progress during build                                                                        |
+| `--help`                | boolean         | Outputs list of supported flags and commands                                                                   |
```

Comment:
```suggestion
| `--help`                | boolean         | Output list of supported flags and commands                                                                   |
```
Let's keep consistency. There're many other similar problems, please update them as well.

Reply:
hmm updating in cli too https://github.com/webpack/webpack-cli/pull/2307